packages/manager/test/SliceMachineManager-versions-getSliceMachineReleaseNotesForVersion.test.ts

```diff
@@ -0,0 +1,157 @@
+import { expect, it } from "vitest";
+
+import { createTestPlugin } from "./__testutils__/createTestPlugin";
+import { createTestProject } from "./__testutils__/createTestProject";
+import { mockGitHubReleasesAPI } from "./__testutils__/mockGitHubReleasesAPI";
+
+import { createSliceMachineManager } from "../src";
+
+it("returns release notes from GitHub for a given version", async (ctx) => {
```

Comment:
The `VersionsManager` methods return only one version at a time. In SM UI, we now request them individually.

In the future, we could optimize the changelog page by only requesting the changelog notes only when a version is selected. This would make start up quicker since we wouldn't need to communicate with GitHub every time SM is refreshed.